platform/device/worker.go

```diff
@@ -179,6 +191,45 @@ func (w *Worker) updateFromCheckout(ctx context.Context, message []byte) error {
 
 }
 
+func (w *Worker) updateFromGetBootstrapToken(ctx context.Context, message []byte) error {
```

Comment:
So I'm not sure there's any need to listen for a _get_ request. Because we need to actively return data there's nothing really to "listen" for here. Indeed it only looks like we update the LastSeen and/or Await state here. I think this can go in lieu of coupling the device service to the check-in service, mentioned elsewhere.

Reply:
I forgot to do anything for this one. I need to check the code again if I figure out how to change it. And if you think that this is still a good idea.